bin/resource_management/update_resource_format.py

```diff
@@ -0,0 +1,31 @@
+import sys
+import csv
+import ckanapi
+"""
+Updates Resources based on incorrect_file_types_report.csv
+Updates Content-type and content-length (if applicable).
+'source.action.resource_patch(id=resource_id,format=new_format)'
+
+Input:
+argv[1]: site URL
+argv[2]: API Key
+argv[3]: CSV Report ('incorrect_file_types_report.csv')
+
+"""
+
+source = ckanapi.RemoteCKAN(sys.argv[1],apikey=sys.argv[2])
+file_report = sys.argv[3]
+file = open(file_report, "r")
+reader = csv.reader(file)

```

Comment:
try https://docs.python.org/3/library/csv.html#csv.DictReader to make the code below more readable